### PR TITLE
Add NLLicensePlateField with tests

### PR DESCRIPTION
**Pull request by jarmovanlenthe**
Added a ModelField and Validator for Dutch License Plates (`NLLicensePlateField`) and a FormField (`NLLicensePlateFormField`) which also sanitizes input.

**All Changes**

- [DONE] Add an entry to the docs/changelog.rst describing the change.

- [DONE] Add an entry for your name in the docs/authors.rst file if it's not
      already there.

- [DONE] Adjust your imports to a standard form by running this command:

      `isort --recursive --line-width 120 localflavor tests`

**New Fields Only**

- [DONE] Prefix the country code to all fields.

- [DONE] Field names should be easily understood by developers from the target
      localflavor country. This means that English translations are usually
      not the best name unless it's for something standard like postal code,
      tax / VAT ID etc.

- [DONE] Prefer '<country code>PostalCodeField' for postal codes as it's
      international English; ZipCode is a term specific to the United
      States postal system.

- [DONE] Add meaningful tests. 100% test coverage is not required but all
      validation edge cases should be covered.

- [DONE] Add `.. versionadded:: <next-version>` comment markers to new
      localflavors.

- [DONE] Add documentation for all fields.

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -10,11 +10,11 @@ New flavors:
 
 New fields for existing flavors:
 
-- None
+- `NLLicensePlateField` in NL flavor.
 
 Modifications to existing flavors:
 
-- None
+- Added `NLLicensePlateField` to NL flavor.
```
Comment (benkonrath) [inline docs/changelog.rst]: The entry is only needed once in the 'New fields for existing flavors' section.

File: localflavor/nl/models.py [hunk 2]
```diff
@@ -82,3 +82,28 @@ def formfield(self, **kwargs):
         defaults = {'form_class': forms.NLBSNFormField}
         defaults.update(kwargs)
         return super(NLBSNField, self).formfield(**defaults)
+
+
+class NLLicensePlateField(models.CharField):
+    """
+    A Dutch car license plate.
```
Comment (benkonrath) [inline localflavor/nl/models.py]: It's probably better to remove 'car' as this can be used for all Dutch license plates.
